### PR TITLE
Finished transcript #11 with agenda headers, readme fully updated

### PR DESCRIPTION
**Pull request by deeayeen**
You can find everything I was unsure about by searching "?" with ctrl/cmd + F.

This one was rough, took me 7 hours

### CODE REVIEW

File: notes/transcript11.md [hunk 1]
```diff
@@ -0,0 +1,585 @@
+# Intro
+
+0:00 - 1:00
+
+Oskar: Welcome to the eleventh Status Core Dev Call. We have quite a lot of things on the agenda today. We did a poll SMT voting to find the things people want to talk the most about. We'll start from the top and see how far we get with the time we have. Also just a small PSA, if you go to the repository, status-im/pm, we have a index of all the previous videos and notes if you want to refer back. Let's get started, so the first topic is Whisper availability inside Status Dapp browser, which is something that will benefit Teller Network as well as Gas Relayer and possibly others. So I guess Richard, would like to elaborate on it if you don't mind?
```
Comment (oskarth) [inline notes/transcript11.md]: SMT -> SNT

File: notes/transcript11.md [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+
+
+# Whisper Availability
+
+1:00 - 2:00
+
+Richard: So basically we are going to do the development of the Gas Relayer and Teller Network. We saw that we have a need to have Whisper available in all the apps, mostly because we want all the apps to be able to communicate between each other. What the rest of the team working on the Gas Relayer and Teller Network and I wanted to know is what is the coordinate state of this functionality and what would it take for Dutch to be able to use Whisper in the (Work Tree?) provider that the Status provides?
```
Comment (oskarth) [inline notes/transcript11.md]: @richard-ramos do you know what this is? (Dutch)

Work Tree -> Web3

File: notes/transcript11.md [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+
+Status (Dutch?): Well since I've been discussing this before, probably since it's building, I have improved my answer. So first question is, do you need this functionality only when the Dapp is open in the status browser or do you need it to send the synchronous notifications to somewhere.
```
Comment (oskarth) [inline notes/transcript11.md]: Status -> Igor (and below), since he was hosting the Zoom call

File: notes/transcript11.md [hunk 4]
```diff
@@ -0,0 +1,585 @@
...
+
+Status (Dutch?): That's why when we should release only when it's generates a separate pair of keys so we know it's a dapp on behalf of the user and not the user himself.
+
+14:00 - 15:00
+
+Iuri: So just a quick question regarding what we just talked about, the central transactions so to speak, you're talking about RPC server and all that, but can't we just, the whisper object that we inject into the dapp, can't we put an api that all it does is forward the package?
+
+Status (Dutch?): Yeah I guess the latest idea was like this. Just generate the entire envelope yourself, the api just sends the envelope already encrypted with what you want. That's what we wanted to essentially, we worked on to enable bots on whisper but it didn't go very far so...
+
+Iuri: We actually have all that in GS already with murmur, and then it's just a matter of the transport protocol. That way the injected object would have entry and murmur can just connect to that.
```
Comment (oskarth) [inline notes/transcript11.md]: GS? @richard-ramos @iurimatias

File: notes/transcript11.md [hunk 5]
```diff
@@ -0,0 +1,585 @@
...
+
+15:00 - 16:00
+
+Iuri: For Teller this would be great, if the browser supports whisper we can use that, if not we could use (lipidTP?) and all that stuff. So we can talk outside the call about how that API could look like.
```
Comment (oskarth) [inline notes/transcript11.md]: lipidTP -> libp2p

File: notes/transcript11.md [hunk 6]
```diff
@@ -0,0 +1,585 @@
...
+
+Ricardo: I think if we have something like the sticker markets and the Croatian markets then having this operation over the sticker packs, we might have a kind of optional fee and maybe when the owner of that sticker pack can specify how much a fee goes to the Croatian market automatically based on the user buys. It goes slowly rising organically as users buy it. And of course if the sticker market artist want to promote their own sticker pack they could just deposit in the Croatian market.
+
+Rachel: I think that there was also a similar suggestion made that artists when they submit their packs to the registry they could optionally donate some percentage of sales to status.
+
+Jarrad: From an implementation standpoint, going back to this as a registry discussion, from an implementation standpoint many of these look very similar. Basically it's a certain number and sending it to a certain address.
+
+31:00 - 32:00
+
+Jarrad: If we're choosing to burn then it's just zero. Zero otherwise the address be like the Eurasian market or whatever. So we can definitely defer this decision-making while not impeding implementation so much, if we just implement these calls.
```
Comment (oskarth) [inline notes/transcript11.md]: Eurasian market seems wrong.

@jarradh @richard-ramos @3esmit do you know? Guessing something like DAO but not sure how that'd parse

File: notes/transcript11.md [hunk 7]
```diff
@@ -0,0 +1,585 @@
...
+
+Jarrad: Optional I imagine, and set it to zero to start with.
+
+Rachel: Yeah I agree.
+
+Ricardo: Ok we will focus in going in that direction in the Sticker Market contract. Maybe we will have some other calls just regarding this sticker market where we make the detail how the user experience over this will be, the optional things.
+
+37:00 - 38:00
+
+Ricardo: Because that should be not only for the (MVV?) but for the whole idea for the final roadmap of sticker market, so take into consideration, we didn't have this decision, and I think everyone mostly agrees with the idea.
```
Comment (oskarth) [inline notes/transcript11.md]: MVV -> MVP

File: notes/transcript11.md [hunk 8]
```diff
@@ -0,0 +1,585 @@
...
+
+Oskar: Alright. So moving on, ULC just got merged upstream after eight months, so that awesome. What does this mean for status and can we integrate it sometime soon? I guess Igor you have the context from the (LESS integration effort?) what's your take?
```
Comment (oskarth) [inline notes/transcript11.md]: LES integration effort

File: notes/transcript11.md [hunk 9]
```diff
@@ -0,0 +1,585 @@
...
+
+38:00 - 39:00
+
+Igor: Well there is even a PR that integrates ULC but as a separate patch that enables it, but the problem with the ULC is the same as the problem with LESS, it needs servers somewhere. And with all this Constantinople thing there are no reliable servers even on the main nor Rinkeby right because as I understand last time it was tested on Rinkeby. So we need to wait until Constantinople is finally resolved. That's the big blocker, and it was since I started integrating this, because we started doing something then half of the nodes were not Constantinople half were Constantinople. Also you'll see is not a magic thing, so it requires that you trust some nodes, and that means that you kinda need some place, and it's not solved, it's just like this, this PR is essentially technical implementation of what if you know, who you can trust, then you can do this quick sink solution something like this.
+
+39:00 - 40:00
+
+Igor: But the question is how do you know who you can trust, and that's a big question.
+
+Oskar: It's still a step up from only offering (Rivera?)
```
Comment (oskarth) [inline notes/transcript11.md]: Rivera -> Infura I think

File: notes/transcript11.md [hunk 10]
```diff
@@ -0,0 +1,585 @@
...
+
+Igor: Sure but that means we also have to host this trusted servers ourselves and protect them.
+
+Oskar: On that note, Andrei Petro, who's working on webnode, he also reached out and he's looking to integrate the ULC support for webnode, so that would be at least one source for these types of nodes. It's definitely not a general solution but it's a potential start maybe.
```
Comment (oskarth) [inline notes/transcript11.md]: webnode -> VIPnode

Andrei Pedro ->  Andrey Petrov.

File: notes/transcript11.md [hunk 11]
```diff
@@ -0,0 +1,585 @@
...
+
+40:00 - 41:00
+
+Igor: Yeah but the problem is that you can't validate anything, because what's different from LAS and ULC is that ULC doesn't validate blocks properly. It sort of defers this thing to other nodes that it "trusts". So where will you get these trusted nodes from. Sources of just blocks, you can find them. But which ones you can trust which ones you can't that's the big question.
```
Comment (oskarth) [inline notes/transcript11.md]: LAS ->LES

File: notes/transcript11.md [hunk 12]
```diff
@@ -0,0 +1,585 @@
...
+
+45:00 - 46:00
+
+Igor: Yeah that I agree and that's why both LESS and ULC are not that high on the roadmap right now because it's a huge project on how to do the whole crypto economy balance that everyone can run nodes and somehow communicate and validate eacho ther, or maybe it's even worse than inferior.
+
+Jarrad: I agree with both the workload and the secondary step, if there's the ability at least to expose the ability to run ULC even if you bring your own node, that would be a great step in the right direction.
+
+46:00 - 47:00
+
+Igor: Yeah that's possible to implement relatively in the short term. I don't know if the PR was merged or how well tested it was. Because we had problems with ADS and ULC probably has the same issues with chain being too far away from what it actually is and doesn't really handle when the chain forks and then remerges, it didn't handle this well. We can try to enable that as test mode when we upgrade status go next time.
```
Comment (oskarth) [inline notes/transcript11.md]: ADS? @mandrigin

File: notes/transcript11.md [hunk 13]
```diff
@@ -0,0 +1,585 @@
...
+
+48:00 - 49:00
+
+Igor: It's the same blockchain code with some thing disabled or delegated to other nodes. So it's not like something on top of blockchain.
+
+Ricardo: Yeah I agree that there is this difference, in Ethereum you have this linked information because of the blockchain, but in stages you don't have that, we need to solve this problem of having a proof of state sending a message, we didn't solve this, so every message is valid while in Ethereum not.
+
+49:00 - 50:00
+
+Oskar: Just on the topic of what differences between this and having your own remote or whatever, if you're running everything yourself it's just one to one and there's no difference, but this also opens up, because if you ULC you can have multiple nodes right, you can have semi trusted set up, maybe you have multiple friends, and lets say you have LESS service in dapp node you can connect to multiple and you don't need to trust anyone but it's still, if you check that 3/5 still return the same hash and you're still better off, so that type of racing you can't do if you just have someone else's remote servers, so it's slightly more sophisticated as well, which I think is the difference.
```
Comment (oskarth) [inline notes/transcript11.md]: LESS -> LES

File: notes/transcript11.md [hunk 14]
```diff
@@ -0,0 +1,585 @@
...
+
+Hester: I'm trying to find my list questions. So the intention would be not to go too much in depth it's more to make an inventory of who is concerned about arbitration as in what swarm will need some form of arbitration. It came up in Teller Network, Sticker Market, dapp store, so it seems to be a relevant topic and I'd like to understand for whom it is most relevant. So I'd like to do a sort of round-the-room to the swarm leads to understand has arbitration come up, are you already working on it, do you see any likely forms of implementation?
+
+51:00 - 52:00
+
+Ricardo: I think the most solid implementation of arbitration that I've seen is (collaterals.io?) from consensus. I'm not sure if it's running but I've seen some presentations, it seems like a good solution. They provide examples of user experience, it's mostly like people trusts random other people to deal with their problems. The majority will select the most obvious answer that is the truth.
```
Comment (oskarth) [inline notes/transcript11.md]: @3esmit do you know what this is?

Comment (oskarth) [reply notes/transcript11.md]: collaterals io

File: notes/transcript11.md [hunk 15]
```diff
@@ -0,0 +1,585 @@
...
+
+Hester: So Teller Network or Sticker Market do you see relevance of arbitration anywhere else?
+
+Julien: In the case of sticker market, as Ricardo said we are considering using decoration market so I'm curious to hear if you think there is some kind of limitation of using decoration market specifically for using sticker market.
+
+Ricardo: It depends on how in the end the decoration market will look like, but in the general sense there is this limitation, because you can not take an action to change something in the decoration market, you can only create, maybe you can create a sticker market over a topic for example, but it kind of gets not consistent with the data set of the sticker market itself that says it has only one category. So the things should make sense in this case.
+
+56:00 - 57:00
+
+Ricardo: So maybe the decoration market would be used in the sticker market but in the sense of the ranking of better stickers, while the arbitration would be used to actually remove a sticker pack from the existence of the dapps. That I think is the difference.
```
Comment (oskarth) [inline notes/transcript11.md]: decoration -> curation

File: notes/transcript11.md [hunk 16]
```diff
@@ -0,0 +1,585 @@
...
+
+1:04:00 - 1:05:00
+
+Jarrad: However if we relicense under CC zero it basically waives everything in the public domain however it degrades into a very liberal license in the event that the law does not allow you to put code into the public domain. So in this case that would allow the community to take the source code and create a new source of truth, in which the GmbH would probably maintain a fork of and contribute to. So that requires the participation of every contributor that who's contributed to the codebase. So basically I wanted to see if there's any objections to that. You can mull it over. But then we have to go rounding up people and getting them to sign some basic paperwork allowing us to do that.
```
Comment (oskarth) [inline notes/transcript11.md]: CC zero -> CC0

**Comment by oskarth** [comment]
Also not sure if you have permissions, but if you can please add me as a reviewer for PR and it'll show up in my GH TODO

File: notes/transcript11.md (hunk 1 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 3 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 2 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 5 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 7 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 8 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 10 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 13 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 15 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

File: notes/transcript11.md (hunk 16 above)
Comment (deeayeen) [reply notes/transcript11.md]: fixed

**Comment by deeayeen** [comment]
> Also not sure if you have permissions, but if you can please add me as a reviewer for PR and it'll show up in my GH TODO

I do not have permissions

**Pull request merged**